components/code-preview.js

```diff
@@ -27,9 +27,9 @@ const StyledLiveError = glamorous(LiveError)((props, {colors, fonts}) => ({
   padding: '1rem',
 }))
 
-const CodePreview = props => {
+const CodePreview = ({noInline = true, code}) => {
   return (
-    <StyledLiveProvider noInline code={props.code} scope={{glamorous}}>
+    <StyledLiveProvider noInline={noInline} code={code} scope={{glamorous}}>
```

Comment:
At this point we might as well pass in the `scope` as well. If we ever want an example that relies on `glamor` or some other package we'll be able to pass that in via `props`.